backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/POServicesFacade.java

```diff
@@ -34,6 +38,25 @@
 	private ITrxManager _trxManager;
 	private ADReferenceService _adReferenceService;
 
+	private static PerformanceMonitoringService _performanceMonitoringService;
+	private static final String PM_METADATA_CLASS_NAME = "PO";
+	private static final String PM_METADATA_SAVE_EX_ACTION = "saveEx";
+	private static final String PM_METADATA_LOAD_ACTION = "load";
```

Comment:
pls inline those 3 constants.... it's quite pointless to have them as constants.
PM_METADATA_CLASS_NAME = "PO";
private static final String PM_METADATA_SAVE_EX_ACTION = "saveEx";
private static final String PM_METADATA_LOAD_ACTION